src/state/recipes/actions.js

```diff
@@ -43,9 +55,9 @@ export function fetchFilteredRecipesPage(pageNumber = 1, filters = {}) {
     if (pageNumber !== Infinity) {
       options.data.page = pageNumber;
     }
-    const recipes = await dispatch(makeNormandyApiRequest(requestId, 'v3/recipe/', options));
+    const recipes = await dispatch(fetcher(requestId, 'v3/recipe/', options));
     while (pageNumber === Infinity && recipes.next) {
-      const nextRecipes = await dispatch(makeApiRequest(requestId, recipes.next));
+      const nextRecipes = await dispatch(fetcher(requestId, recipes.next));
```

Comment:
this should still use `makeApiRequest`. it will use the correct url because `recipes.next` is set by the correct server already.

Reply:
Oops. That's definitely an oversight. I believe I managed to *not* do that in the other functions that mix the two types of request concepts.